iam-login-service/src/main/java/it/infn/mw/iam/api/aup/AupSignaturePageController.java

```diff
@@ -126,6 +131,12 @@ public ModelAndView signAup(HttpServletRequest request, HttpServletResponse resp
 
     return new ModelAndView("redirect:/dashboard");
   }
+
+  @ResponseStatus(value = HttpStatus.METHOD_NOT_ALLOWED)
+  @ExceptionHandler(IamAupSignatureUpdateError.class)
+  public ErrorDTO aupSignatureUpdateError(Exception ex) {
```

Comment:
In this case, it would be better to return the error message as a JSON response?
```
@ExceptionHandler(IamAupSignatureUpdateError.class)
  public ResponseEntity<ErrorDTO> aupSignatureUpdateError(Exception ex) {
      ErrorDTO errorResponse = ErrorDTO.fromString(ex.getMessage());
      return new ResponseEntity<>(errorResponse, HttpStatus.METHOD_NOT_ALLOWED);
  }
```

Reply:
Thanks. Done.